### PR TITLE
NDSForwarder -> YANBF

### PR DESCRIPTION
**Pull request by lifehackerhansol**
Yeah I'm self-advertising.

To be completely honest I think just for the sake of simplicity we should keep NDSForwarder on there, but I don't know. The instructions are a complete mess (though less of a mess than Forwarder3-DS).

### CODE REVIEW

**Comment by lifehackerhansol** [comment]
lmfao

-7 lines, +25 lines

yeah ndsforwarder is too ez

File: pages/_en-US/ds-index/forwarders.md [hunk 1]
```diff
@@ -35,13 +35,31 @@ If you already have Universal Updater installed on your console, you can skip to
...
+
+### Part 2: Generating forwarder titles
+1. Power off your console
+1. Insert your SD card into your computer
+1. Install [Python 3](https://python.org/downloads) for your computer
+1. Go to [YANBF releases page](https://github.com/lifehackerhansol/YANBF/releases) and download the respective `.zip` file for your computer's OS
+1. Extract the `.zip` file you downloaded
+1. Open a terminal (or Command Prompt), then `cd` to the `generator` folder
+1. .
+    - Windows users: run `pip install -r requirements.txt`
```
Comment (Epicpkmn11) [inline pages/_en-US/ds-index/forwarders.md:49]: Don't you need to do
```
py -3 -m pip install -r requirements.txt
```
on Windows?

File: pages/_en-US/ds-index/forwarders.md [hunk 2]
```diff
@@ -35,13 +35,31 @@ If you already have Universal Updater installed on your console, you can skip to
...
+    - Others: run `pip3 install -r requirements.txt`
+1. For macOS users:
+    1. Open the YANBF GUI `.dmg` file
+    1. Copy your app to the `generator` folder
+1. Double click on the GUI application to open the generator
```
Comment (Epicpkmn11) [inline pages/_en-US/ds-index/forwarders.md:54]:
```suggestion
    1. Copy the app to your Applications folder
1. Open the GUI application
1. Set the `Generator Directory` on the right to the `generator` folder
```
The first change is fine either way I guess, installing to Applications makes more sense imo as that's more normal and the DMG even tells you to, but I guess either's fine.

Second line is just a bit oddly specific imo, it's not always even a double click to open an application.

The final change could probably be worded better, but at least on macOS you need to set that manually. Not sure if you need to on other OSes.

File: pages/_en-US/ds-index/forwarders.md [hunk 3]
```diff
@@ -35,13 +35,31 @@ If you already have Universal Updater installed on your console, you can skip to
...
+1. Drag and drop the ROM **on your SD card** to the app
```
Comment (Epicpkmn11) [inline pages/_en-US/ds-index/forwarders.md:55]:
```suggestion
1. Drag and drop the ROM **from your SD card** to the app
```

File: pages/_en-US/ds-index/forwarders.md [hunk 4]
```diff
@@ -35,13 +35,31 @@ If you already have Universal Updater installed on your console, you can skip to
...
+1. Click on the Download icon
+1. Your forwarder `.cia` will be generated to a newly created `output` folder
+1. Copy the `.cia` file to your SD card
+1. Reinsert your SD card into your console
+1. Power on your console
+1. Launch FBI
+1. Navigate to `SD -> <your .cia file>`, select it, then select `Install and delete`
```
Comment (Epicpkmn11) [inline pages/_en-US/ds-index/forwarders.md:62]:
```suggestion
1. Navigate to `SD` -> `<your .cia file>`, select it, then select `Install and delete`
```

**Review by Epicpkmn11 — COMMENTED** [review]
just a couple minor things

File: pages/_en-US/ds-index/forwarders.md [hunk 5]
```diff
@@ -35,13 +35,31 @@ If you already have Universal Updater installed on your console, you can skip to
...
-1. Open the Homebrew Launcher
-1. In the Homebrew Launcher, open `NDS Forwarder Generator`
-1. Navigate to your game's location and press <kbd class="face">A</kbd>
+1. Install the YANBF package
+1. YANBF and its required files are now set up in its respective locations
+
+### Part 2: Generating forwarder titles
+1. Power off your console
+1. Insert your SD card into your computer
+1. Install [Python 3](https://python.org/downloads) for your computer
```
Comment (LightSage) [inline pages/_en-US/ds-index/forwarders.md:44]: Probably should be specific with this, i.e. Python 3.6+

**Comment by lifehackerhansol** [comment]
moot for now

**Pull request closed without merging**